lib/pathutils.c

```diff
@@ -92,3 +93,87 @@ get_filename_extension(const gchar *filename)
 
   return start_ext + 1;
 }
+
+GPtrArray *
+get_filenames(const gchar *dir_path, gboolean recursive, gchar *pattern)
```

Comment:
Should the `libsyslog-ng` be extended with those ?
This pattern can be found in plugin listing, wildcard file monitor, config include. Moving this into a common place I would expect to do code reuse; at least one reason could be. In this case I would not do that.

But rather keep it in the scope of `pdbtool`, which I would prefer.

Reply:
I agree, this will be moved.

Reply:
I moved it :)